imports/ui/components/atoms/AlertDanger.svelte

```diff
@@ -0,0 +1,35 @@
+<!-- 
+  I ask you guys to give it a cool translation animation on display 
+  It looks like it comes from nowhere...
+-->
+
+<script>
+  export let primaryText = 'Text one';
+  export let secondaryText = 'Text two';
+  export let open = false;
+</script>
+
+
+<div 
+  class="{open ? 'flex' : 'hidden'} justify-center w-full bottom-2 fixed"
```

Comment:
you could have used the sintax `class:flex={open}`. it would make the code clearer